test/chrome_test.js

```diff
@@ -62,9 +64,7 @@ describe("StorageArea", () => {
       }).catch(done);
     });
 
-    it("set/remove a value in IDB", (done) => {
-      const area = new StorageArea("http://localhost:8080/v1/", "sync");
-
+    it("set/remove a value in IDB", function(done) {
       area.set({"something": 1}, function () {
         area.remove("something", function() {
           area.get("something", function(items) {
```

Comment:
We could use promisify to write the tests for the callbacks-based storage.sync API, but I doubt it would make them more readable.

In the code, we trigger these callbacks from the promises returned by Kinto, so there we would benefit from using some sort of callbackify :) e.g. https://github.com/Kinto/kinto-chrome/blob/master/lib/chrome.js#L115-L117